src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidation.java

```diff
@@ -23,7 +21,7 @@
 	public LocateMatchingApplicantsResponse locateMatchingApplicants(LocateMatchingApplicantsRequest lmr)
 			throws FigaroValidationServiceException;
 	
-	public ValidateApplicantServiceResponse validateApplicantService(ValidateApplicantServiceRequest lmr)
+	public ValidateOrgApplicantServiceOrdsResponse validateOrgApplicantServiceOrdsResponse(String applPartyId,String orgPartyId)
```

Comment:
Please change to ValidateApplicantServiceOrdsResponse. Picky I know, but it makes things easier. Thanks

Reply:
renamed the class